library/build.gradle

```diff
@@ -31,6 +31,9 @@ task jar(type: Jar) {
 
 dependencies {
     compile 'com.android.support:support-annotations:23.3.0'
+    compile 'com.android.support:appcompat-v7:25.3.0'
```

Comment:
is there a problem with having different versions of support libraries in the same project?  Seems like keeping them in sync is the better way to go.